frontend/apps/filemanager/filemanager.lua

```diff
@@ -1551,8 +1551,11 @@ end
 function FileManager:openFile(file, provider, doc_caller_callback, aux_caller_callback)
     if provider == nil then
         provider = DocumentRegistry:getProvider(file, true) -- include auxiliary
+    else
+        provider.forced = true
```

Comment:
(Can't guess after a quick glance at the code) Is this `provider` object a global/persistent one, or some object created at this document opening time. If persistent, it's a little ugly to add to it a `forced=true` property, that may leak to next usages and would then need maintenance/resetting.

Reply:
> then need maintenance/resetting

Global, to avoid passing it through as arg.
Sure, I do, for aux provider
https://github.com/koreader/koreader/blob/b6e5138faaa5601ceb566842ec4b2f47f0fcb06b/frontend/apps/filemanager/filemanager.lua#L1558
and for document provider
https://github.com/koreader/koreader/blob/b6e5138faaa5601ceb566842ec4b2f47f0fcb06b/frontend/apps/reader/readerui.lua#L654

Reply:
(Still feels like bad practice.)

Reply:
Thanks for the change :)